controller/bookController.go

```diff
@@ -0,0 +1,47 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/joseantoniovz/academy-go-q12021/service"
+)
+
+func GetBook(w http.ResponseWriter, r *http.Request) {
+	var books, err = service.GetAll()
+	if err != nil {
```

Comment:
Usually you would like to log the error that you're getting from your services and return another custom error that doesn't expose the functionality of your api to the user